src/rules/adaptive-mfa-custom-confidence.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * @title Adaptive MFA with Custom Confidence
+ * @overview Trigger multifactor authentication for a specific risk assessment result.
+ * @gallery true
+ * @category multifactor
+ *
+ * This rule is used to trigger multifactor authentication when a specific risk assessment result is detected.
+ *
+ * The `context.riskAssessment` attribute will be available only when the Adaptive MFA policy is enabled for your tenant. This feature is only available to customers that have subscribed to an Enterprise plan.
```

Comment:
FIXME: copy the sentence about Enterprise plan to a js doc inside the rule templates.

Reply:
And use the copy in this ticket as a comment: https://auth0team.atlassian.net/browse/CAUTH-611